lib/vmdb/loggers/formatters/container.rb

```diff
@@ -0,0 +1,47 @@
+class ContainerFormatter < Logger::Formatter
+  SEVERITY_MAP = {
+    "DEBUG"   => "debug",
+    "INFO"    => "info",
+    "WARN"    => "warning",
+    "ERROR"   => "err",
+    "FATAL"   => "crit",
+    "UNKNOWN" => "unknown"
+    # Others that don't match up: alert emerg notice trace
+  }.freeze
+
+  def call(severity, time, progname, msg)
+    # From https://github.com/ViaQ/elasticsearch-templates/releases Downloads asciidoc
+    {
+      :@timestamp => format_datetime(time),
+      :hostname   => hostname,
+      :level      => translate_error(severity),
+      :message    => build_message(msg),
+      :pid        => $PROCESS_ID,
+      :tid        => Thread.current.object_id,
+      :service    => progname,
+      # :tags => "tags string",
+    }.to_json << "\n"
+  end
+
+  private
+
+  def build_message(msg)
+    msg = msg2str(msg)
+
+    # Add task id to the message if a task is currently being worked on.
+    if $_miq_worker_current_msg && !$_miq_worker_current_msg.task_id.nil?
+      prefix = "Q-task_id([#{$_miq_worker_current_msg.task_id}])"
+      msg = "#{prefix} #{msg}" unless msg.include?(prefix)
+    end
```

Comment:
Can we piggyback on the existing formatter to do this for us?  Or perhaps change the VmdbLogger::Formatter to piggyback on this?  I don't really like that this has to be duplicated.